src/components/CharacterForm/CharacterForm.js

```diff
@@ -98,6 +99,47 @@ const CharacterForm = ({ setOpen }) => {
     )
   }
 
+  const submit = handleSubmit(onSubmit, (errors) => {
+    setErrors(errors)
+  })
+
+  const rules = {
+    name: {
+      required: "This field is required",
+      minLength: {
+        value: 3,
+        message: "This field should contain at least 3 characters",
+      },
+      pattern: {
+        value: /^[^\s]{3}.*/,
+        message: "The first 3 characters shouldn't be space!",
+      },
```

Comment:
This rule isn't what we want, we want to allow spaces anywhere the user wants, but be don't want to accept input that is only space characters, there must be at least 3 non space characters in the name of the actor. Sorry if I wasn't clear about that before